base/src/com/google/idea/blaze/base/sync/projectstructure/ModuleEditorImpl.java

```diff
@@ -120,7 +119,7 @@ public void commitWithGc(BlazeContext context) {
     List<Module> orphanModules = Lists.newArrayList();
     for (Module module : ModuleManager.getInstance(project).getModules()) {
       if (!modules.containsKey(module.getName())
-              && (ModuleValidator.EP_NAME.getExtensionList().isEmpty() || ModuleValidator.EP_NAME.getExtensionList().stream().anyMatch(n -> !n.isModuleValid(module)))) {
+              && (ModuleValidator.EP_NAME.getExtensionList().isEmpty() || ModuleValidator.EP_NAME.getExtensionList().stream().anyMatch(n -> !n.hasOtherContentRoot(module)))) {
```

Comment:
Instead of 
`ModuleValidator.EP_NAME.getExtensionList().stream().anyMatch(n -> !n.hasOtherContentRoot(module)))`
it should be `!ModuleValidator.EP_NAME.getExtensionList().stream().anyMatch(n -> n.hasOtherContentRoot(module)))` or so.